### PR TITLE
Kcoley/vec3normalized fixes

### PR DESCRIPTION
**Pull request by kcoley**
Adds support for Vector3 Vertex Colors, normalized byte and shorts for colors and texture coordinates for the glTF loader (2.0).

### CODE REVIEW

**Comment by deltakosh** [comment]
Hello I'm sorry but there is a small conflict

**Comment by kcoley** [comment]
I added some logic to enable hasVertexAlpha when COLOR_0 is being used.

**Comment by deltakosh** [comment]
Was it test with the visual validation suite?

**Comment by kcoley** [comment]
Looks to be some visual differences in the visual validation suite.  I'll see if I can figure out why:

visual_validation_results

**Comment by kcoley** [comment]
This update now does not enable vertexcolor in the glTF 2.0 loader due to some needed alpha changes.

File: loaders/src/glTF/2.0/babylon.glTFLoader.ts [hunk 1]
```diff
@@ -474,6 +474,76 @@ module BABYLON.GLTF2 {
             }
         }
 
+        /**
+         * Converts a data bufferview into a Float4 Texture Coordinate Array, based on the accessor component type
+         * @param {ArrayBufferView} data 
+         * @param {IGLTFAccessor} accessor 
+         */
+        private _convertToFloat4TextureCoordArray(data: ArrayBufferView, accessor: IGLTFAccessor): Float32Array {
+            const buffer = <Float32Array>data;
```
Comment (bghgary) [inline loaders/src/glTF/2.0/babylon.glTFLoader.ts]: buffer is not always a Float32Array. not sure what the consequences of that is. should probably cast to the correct type based on componentType.

**Comment by deltakosh** [comment]
Sorry but my last change add some conflicts :(

**Comment by kcoley** [comment]
No worries, working on the fix.

**Pull request merged**